docs/content/faq.md

```diff
@@ -58,3 +58,37 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
 ## HedgeDoc fails executing migrations and does not start
 Unfortunately, older versions of HedgeDoc had some bugs regarding migrations and didn't always record that a migration was executed.  
 Have a look at the *[Troubleshooting Migrations](/guides/migration-troubleshooting/)* guide for more information.
+
+## Why does my interface look weird?
+
+If your interface looks like this
+
+![Broken HedgeDoc interface](./faq/interface_broken.png)
+
+instead of this
+
+![Not broken HedgeDoc interface](./faq/interface_okay.png)
+
+then you should check the following settings (or their corresponding environment variables if you e.g. use docker.):
+
+- `domain`
+- `protocolUseSSL`
+- `urlAddPort`
+
+Check [the configuration page](/configuration/) for an explanation.
+
+These settings control how HedgeCoc generates links to other pages and to assets, therefore you must set them from the outside point of view.
+
+Example: You access your HedgeDoc instance directly via `http://93.184.216.34:5812`. You must set:
+
+- `domain` to `93.184.216.34` because that's the host.
+- `protocolUseSSL` to `false` because you access your instance via HTTP instead of HTTPS.
+- `urlAddPort` to `true` because you access the instance using a custom port instead of the default HTTP port 80.
+
+Example: You access your HedgeDoc instance using a reverse proxy via `https://markdown.pro`. You must set:
```

Comment:
Use `.example` as tld for the domain. This is explicitly reserved for examples and will never be assigned.

See the [RFC 2606](https://datatracker.ietf.org/doc/html/rfc2606#section-2) for this.